@xen-orchestra/proxy/docs/api.md

```diff
@@ -139,10 +139,14 @@ declare namespace backup {
 
   function importVmBackup(_: { backupId: string; remote: Remote; srUuid: string; xapi: Xapi }): string
 
+  function listPoolMetadataBackups(_: { remotes: { [id: string]: Remote } }): object
```

Comment:
Are the backups grouped by remotes like `listVmBackups`?

If that's the case, please make it clear in the return type :slightly_smiling_face: